src/main/scala/dxWDL/exec/WfOutputs.scala

```diff
@@ -15,7 +15,8 @@ import wom.types._
 import dxWDL.base.{Utils, Verbose}
 import dxWDL.util._
 
-case class WfOutputs(wf: WorkflowDefinition,
+
+abstract class WfOutputsBase(wf: WorkflowDefinition,
```

Comment:
Do we really need the abstract class? I have avoided using `abstract` everywhere else.

Reply:
Got it